apply/index.html

```diff
@@ -53,7 +53,7 @@ <h2>I submitted my application! Now what?</h2>
 			<p>Once you apply, a studio member will check out your application and see if you meet the requirements. If you don't, you will be rejected, in which case you can re-apply in a week. If your application contains all the required components, and you meet the requirements above, a member will post the link in the Forum Helpers Special Actions studio and it will enter a one week "Application Pending" window. During this time, your application will be discussed by current Forum Helpers. If your application gets less than four objections during this time, then your application will be accepted and you will be invited to the studio. If your application gets four or more objections during this time, then your application will be declined and you will be given some feedback on how to improve. If your application is declined, please wait at least one week before reapplying, and try to work on the feedback that you were given before reapplying.</p>
 
 			<h2>I've been accepted! Now what?</h2>
-			<p>Congratulations! You will know that you have been accepted because you will recieve an invite to the studio and a comment on your profile with the good news. Next, please leave a biography (500 characters or less) on the <a href="https://scratch.mit.edu/users/TFH-Bios/" class="FHULink" target="_blank">@TFH-Bios</a> account so that it can be added to this website. Welcome to The Forum Helpers!</p>
+			<p>Congratulations! You will know that you have been accepted because you will receive an invite to the studio and a comment on your profile with the good news. Within a week of your acceptance, you should gain access to <a href="https://theforumhelpers.herokuapp.com/" class="FHULink" target="_blank">The Forum Helpers Dashboard</a>, which you can use to change your biography on the <a href="https://theforumhelpers.github.io/forumhelpers/" class="FHULink" target="_blank">list of Forum Helpers</a> page. If you have any issues with this process or questions about the dashboard, please contact either <a href="https://scratch.mit.edu/users/-Accio-/" class="FHULink" target="_blank">@-Accio-</a> or <a href="https://scratch.mit.edu/users/leahcimto/" class="FHULink" target="_blank">@leahcimto</a> on Scratch. If you have not received access to the dashboard yet, please wait a week before contacting admins about being added.</p>
```

Comment:
Having the wait a week part at the end seems kind of awkward. Also, saying "before contacting admins" might not be helpful because it doesn't specify anywhere that leahcimto and I are admins.

I think two changes could fix this:
1. Change the sentence about issues to:
If you have any issues with this process or questions about the dashboard, please contact a dashboard admin (-Accio- or leahcimto) on Scratch.
2. Change the last sentence to:
If you have not received access to the dashboard by one week after your acceptance date, please contact a dashboard admin.

Reply:
Sure. I'll implement it when I have time.

Reply:
@Accio1 Implemented. Can you take a look at it and let me know what you think of it?